modules/create-perf-profile-workload-partitioning.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: CONCEPT
+[id="performance-profile-workload-partitioning_{context}"]
+= Performance profiles and workload partitioning
```

Comment:
New modules still missing the list of related assemblies in [metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata):
```suggestion
// Module included in the following assemblies:
//
// * scalability_and_performance/enabling-workload-partitioning.adoc

:_mod-docs-content-type: CONCEPT
[id="performance-profile-workload-partitioning_{context}"]
= Performance profiles and workload partitioning
```